### ISSUE TITLE
[Accessibility: ACC-270.1] fd-table not accessible with keyboard tab in a dialog box with other components.

### ISSUE THREAD

**Issue by ShubhamNarendraSingh** [SAP/fundamental-ngx#12775]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
The tab accessible should work in dialog box while accessing a add dialog with keyboards accessible.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [Table](https://sap.github.io/fundamental-ngx/#/core/table)
1.Create a component based dialog that has complex body with different ngx components with table and add more row button.
2.Access each component with tab key the accessibility is lost for table element and directly moves to add more row button.
<img width="1466" alt="Screenshot 2024-11-26 at 3 21 17 PM" src="https://github.com/user-attachments/assets/3a0f3c2c-1164-4bc1-b8d3-7185f5809a09">

#### Please provide stackblitz example(s).
You can try ngx examples, issue is reproducible there.

#### Did you check the documentation and the API?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
closing as there was no activity
